script/backup.sh

```diff
@@ -1,52 +1,70 @@
 #!/bin/bash
-if [ "$(id -u)" != "0" ]; then
-  echo "This script must be run as root" 1>&2
+
+usage(){
+  echo "Usage: katello-backup <backup directory>"
+  echo " eg: $ katello-backup /var/tmp/katello-backup/"
+}
+
+if [ "$#" -ne 1 ]; then
+  echo "ERROR: Please specify an export directory"
+  usage
   exit 1
 fi
 
-BDIR=$(readlink -f "$1")
-mkdir -p $BDIR
+# Backup dir
+umask 0027
+export BDIR=$1katello-backup-`date "+%Y%m%d%H%M"`
+mkdir $BDIR
+chgrp postgres $BDIR
+chmod 770 $BDIR
 cd $BDIR
+echo "Creating backup folder $BDIR"
+echo ""
 
+# Conf files
 echo "Backing up config files... "
 tar --selinux -czvf config_files.tar.gz \
```

Comment:
We lay down some tomcat configuration for Candlepin that is outside of `/etc/candlepin` that we may need.